manim/constants.py

```diff
@@ -259,6 +261,10 @@
 DEFAULT_QUALITY: str = "high_quality"
 DEFAULT_QUALITY_SHORT = QUALITIES[DEFAULT_QUALITY]["flag"]
 
+DEFAULT_RENDER_NOTE: str = (
+    "NOTE: `manim render` is called, if you just type `manim`."
+    "So type `manim render --help` if you'd like to know what the `-qlp` flags do for example."
```

Comment:
```suggestion
    "NOTE: The subcommand 'manim render' is called if no subcommand is specified. "
    "Run 'manim render --help' if you would like to know what the -ql or -p flags do, for example."
```

Does the first line need a trailing space so that the string is rendered with a space between the full stop at the end of the first line and the sentence starting on the second line? Also, no need for backtick inline code highlighting, as the message is "just" displayed in the terminal.